client/my-sites/themes/theme-showcase.jsx

```diff
@@ -58,6 +60,8 @@ const optionShape = PropTypes.shape( {
 	action: PropTypes.func,
 } );
 
+const ShowcaseBanner = sample( [ PhotoBlogBanner, SmallBusinessBanner ] );
```

Comment:
There's one more thing that occurred to me: The Theme Showcase is server-side rendered. Adding this random element will cause reconciliation errors whenever client-side rendering ends up picking the 'other' banner :slightly_frowning_face:

Reply:
As it goes with reconciliation errors, we can't also just skip the banner when rendering on the server -- since that will _also_ cause a renconc error 😄 

One option would be to only render the banner when logged-in, since we don't SSR there. That would fix the reconc issue, but it would also mean that logged-out users would not get to see the banner at all -- which might be undesired behavior.

Reply:
Current issue (seen on http://calypso.localhost:3000/themes while logged-out):
![image](https://user-images.githubusercontent.com/96308/36112850-d35d9356-1022-11e8-9bb6-6c7e8dc780bf.png)